mlem/api/commands.py

```diff
@@ -202,14 +189,12 @@ def clone(
         target,
         fs=target_fs,
         project=target_project,
-        index=index,
-        external=external,
     )
 
 
 def init(path: str = ".") -> None:
-    """Creates .mlem directory in `path`"""
-    path = posixpath.join(path, MLEM_DIR)
+    """Creates mlem config in `path`"""
+    path = posixpath.join(path, MLEM_CONFIG_FILE_NAME)
```

Comment:
I know the file is called `.mlem.yaml`. But for GTO it's `.gto`. Should we rename it to `.gto.yaml` in GTO, or should we rename `.mlem.yaml` in `.mlem`? We shouldn't confuse users with different approaches. Did we had same files in other tools maybe? cc @shcheklein

Reply:
@mike0sv if everything else is good, let's merge this so we won't get a PR queue again? And then discuss it separately.

Reply:
my 2 cents - if the file is indeed a `yaml`, `gto.yaml` > `[something.]gto`
extensions are for formats :)
as long as `gto` doesn't invent a new file format/structure but respects `yaml`, that's the [least surprising](https://en.wikipedia.org/wiki/Principle_of_least_astonishment) thing to do

Reply:
Ok, then it's decided I guess. Let's keep it here, and I'll update it in GTO as well some time later.